pyOCD/flash/flash_klxx.py

```diff
@@ -74,14 +77,21 @@
                'pc_erase_sector' : 0x2000006F,
                'pc_program_page' : 0x2000009F,
                'begin_stack' : 0x20000800,
-               'begin_data' : 0x20000a00,
+               'begin_data' : 0x20000a00,       # Analyzer uses a max of 1 KB data (256 pages * 4 bytes / page) 
+                                                # Note: 128 pages on KL25 and KL26, 256 pages on KL46
                'static_base' : 0x20000000 + 0x20 + 0x5E8,
-               'page_size' : 1024
+               'page_size' : 1024,
+               'analyzer_supported' : False     # Not enough space on KL02 and KL05
               };
 
 # @brief Flash algorithm for Kinetis L-series devices.
 class Flash_klxx(Flash_Kinetis):
 
     def __init__(self, target):
-        super(Flash_klxx, self).__init__(target, flash_algo)
+        algo = flash_algo
+        if isinstance(target, KL25Z) or isinstance(target, KL26Z) or isinstance(target, KL46Z):
```

Comment:
The KL02 and KL05 didn't have enough space to support the analyzer so this is missing from the default flash_algo.  If the target is a KL25Z, KL26Z or KL46Z then analyzer support is added to flash_algo.